index.d.ts

```diff
@@ -365,6 +365,15 @@ declare module "moleculer-web" {
 
 	type ETagFunction = (body: any) => string
 	type AliasFunction = (req: IncomingRequest, res: GatewayResponse) => void
```

Comment:
It has a third param a `next` callback function (to be compatible with express middleware) which accepts an `err` optional parameter